c_emulator/riscv_sim.c

```diff
@@ -298,6 +299,11 @@ char *process_args(int argc, char **argv)
       fprintf(stderr, "enabling storing illegal instruction bits in mtval.\n");
       rv_mtval_has_illegal_inst_bits = true;
       break;
+    case 'o':
```

Comment:
It's now possible to use long-only options (see e.g. OPT_TRACE_OUTPUT), I wouldn't use one of the few remaining short options for such a rarely used one.

Reply:
Ah thanks, I missed that! Looks like the C compiler could do with having all of its flags changed to match the OCaml one.